apps/src/templates/census2017/SchoolAutocompleteDropdownWithLabel.jsx

```diff
@@ -87,7 +88,7 @@ export default class SchoolAutocompleteDropdownWithLabel extends Component {
             schoolFilter={this.props.schoolFilter}
           />
           <label>
-            <input id="schoolNotFoundCheckbox" type="checkbox" onChange={this.handleSchoolNotFoundCheckbox.bind(this)} checked={this.props.value === "-1"}/>
+            <input id="schoolNotFoundCheckbox" type="checkbox" onChange={this.handleSchoolNotFoundCheckbox.bind(this)} checked={schoolNotFound ? true : false}/>
             <span style={styles.checkboxOption}>
```

Comment:
could just be `checked={schoolNotFound}` ?

Reply:
When I did it that way I was getting errors about changing from and uncontrolled to a controlled component when the box got checked. This way fixed that.

Reply:
The issue is that when `this.props.schoolDropdownOption` is `undefined` and `this.props.value` is not `-1` then `schoolNotFound` evaluates to `undefined` rather than `false`.

Reply:
Or `checked={!!schoolNotFound}` if you really care about passing a boolean.